x5.md

```diff
@@ -90,19 +53,18 @@ This can be done in a the following ways:
 #### Contact Request
 
 #### Bundles
-- NOTE: cant seem to find bundle code on account generation
 - An X3DH prekey bundle is defined as ([code(https://github.com/status-im/status-go/messaging/chat/protobuf/encryption.pb.go)]):
   ```
   Identity                // Identity key
   SignedPreKeys           // a map of installation id to array of signed prekeys by that installation id
   Signature               // Prekey signature
   Timestamp               // When the bundle was lasted created locally
-  XXX_NoUnkeyedLiteral    // ???
-  XXX_unrecognized        // ???
-  XXX_sizecache           // ???
   ```
   - include BundleContainer???
 - a new bundle SHOULD be created at least every 12 hours
+- a bundle is only generated when it is used
+- a bundle MUST be distributed on the contact code channel (NOTE: define this where?)
```

Comment:
Can we leave comments in markdown comment format? To make sure the spec is readable as is

<!-- COMMENT -->